include/api/wasmedge/wasmedge.h

```diff
@@ -2636,6 +2658,13 @@ WASMEDGE_CAPI_EXPORT extern WasmEdge_MemoryInstanceContext *
 WasmEdge_CallingFrameGetMemoryInstance(const WasmEdge_CallingFrameContext *Cxt,
                                        const uint32_t Idx);
 
+/// Suspend the execution from the current calling frame.
+/// This is only available when using WasmEdge_Coroutine.
+///
+/// \param Cxt the WasmEdge_CallingFrameContext.
+WASMEDGE_CAPI_EXPORT extern void
```

Comment:
Perhaps developers working on the host section need to know which module's host function triggered the suspend. Therefore, I hope that it can return a pointer to the module instance to which the triggering host function belongs.